### ISSUE TITLE
[Bug]-[724.7]:Focus outline not visible on certain widgets. Eg

### ISSUE THREAD

**Issue by danciaclara** [appsmithorg/appsmith#11602]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Pressing tab key does not focus around certain widgets like buttons, icon buttons, etc

### Steps To Reproduce

Add a few widgets on the form- input, select, button, icon button
Press tab key and you will notice that it does not highlight buttons or icon buttons

### Environment

Production

### Version

Cloud

**Comment by dilippitchika** [comment]
Will be addressed as part of widget reskinning

**Comment by dilippitchika** [comment]
Not addressing this in widget reskinning, will work on a new accessibility epic and pick this

**Comment by jsartisan** [comment]
## Stats
| Stat | Values |
| ------------- | ------------- |
| Reach     | 362.35 |
| Effort (months)  | 0.5 |
